apps/src/blockly/constants.js

```diff
@@ -11,11 +11,15 @@ export const MenuOptionStates = {
 export const Themes = {
   MODERN: 'modern',
   DARK: 'dark',
-  MUSICLAB_DARK: 'musiclabdark',
-  HIGH_CONTRAST: 'highContrast',
+  HIGH_CONTRAST: 'highcontrast',
   PROTANOPIA: 'protanopia',
   DEUTERANOPIA: 'deuteranopia',
-  TRITANOPIA: 'tritanopia'
+  TRITANOPIA: 'tritanopia',
+  MUSICLAB_DARK: 'musiclabdark',
+  MUSICLAB_HIGH_CONTRAST: 'musiclabhighcontrast',
+  MUSICLAB_PROTANOPIA: 'musiclabprotanopia',
+  MUSICLAB_DEUTERANOPIA: 'musiclabdeuteranopia',
+  MUSICLAB_TRITANOPIA: 'musiclabtritanopia'
```

Comment:
Would it make sense to name these a bit more generically so that other labs could have a dark mode and get this same functionality?

Reply:
Not at this point, I don't think. They are named this way because they're really only compatible with MusicLab's overall design. I'd like to reconcile those differences soon that we only need one set of themes and they're compatible with any lab.

Reply:
Here's Dance Party with one of the dark/accessible MusicLab themes. 
<img width="1205" alt="image" src="https://user-images.githubusercontent.com/43474485/223859281-b12c6376-4e13-4ea8-9cd1-11fbcf63d0b8.png">